mobile.css

```diff
@@ -1678,4 +1678,688 @@ button:active {
   .desk4 {
     display: none;
   }
+
+  body {
+    left: 0;
+    box-shadow: none;
+    height: 5461px;
+  }
+
+  .contact-form {
+    height: 500px;
+  }
+}
+
+@media screen and (min-width: 500px) {
+  body {
+    position: relative;
+    width: 1330px;
+    height: 4651px;
+    display: grid;
+    left: 0;
+    box-shadow: none;
+  }
+
+  .main-nav {
```

Comment:
The margin from left and right should be the same on the toolbar. Please see the screenshot below and fix the spacing.